src/action-button/index.js

```diff
@@ -18,6 +20,17 @@ const ActionButton = React.createClass({
             toggle: false,
         };
     },
+
+    _onClick() {
+        if (this.props.linkTo) {
+            window.open(this.props.linkTo, '_self')
+        }
+
+        if (typeof this.props.onClick === 'function') {
```

Comment:
Since we're explicitly using this prop now, it should be specified in propTypes. 

You also need to add the event prop to the function. Since you're overriding onClick, your new method needs to be identical to the old implementation.